app/Website/routes/website-routes.php

```diff
@@ -15,3 +16,14 @@
 Route::get('/blog-post-tags/{tag}', [BlogPostTagsController::class, 'show'])->name('blog-post-tags.show');
 
 Route::post('/contact', ContactController::class)->name('contact')->middleware('throttle:5,1');
+
+Route::post('newsletter/store', [NewsletterSubscribeController::class, 'store'])->middleware('throttle:5,1');
+
+//Which way to be used the above or the bellow?
+
+// Route::middleware('throttle:5,1')->group(function () {
```

Comment:
Please remove the commended code and the comment above.

Reply:
Done